Snail/Variable.swift

```diff
@@ -34,12 +35,13 @@ public class Variable<T> {
     public func bind(to variable: Variable<T>) {
```

Comment:
I think this function should instead return the subscription so we can get rid of the disposer. Callers of this function would then handle the subscription